Assets/Scripts/Model/Types/TurnMode.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace Fumbbl.Model.Types
+{
+    public class TurnMode : FfbEnumerationFactory
+    {
+        public bool StoreLast;
+
+        public TurnMode(string name) : base(name) { }
+
+        public static TurnMode Regular = new TurnMode("regular") { StoreLast = true };
```

Comment:
Make StoreLast default to true in the constructor and only override it in these statics when needed. It's unecessarily verbose to specify it in all these static definitions given that it's so rarely set to false.